advanced-scraping-guide.mdx

```diff
@@ -430,6 +430,12 @@ When using the `/crawl` endpoint, you can customize the crawling behavior with r
 - **Description**: This option allows the crawler to follow links that point to external domains. Be careful with this option, as it can cause the crawl to stop only based only on the`limit` and `maxDepth` values.
 - **Default**: `false`
 
+### `delay`
+
+- **Type**: `number`
+- **Description**: Delay in seconds between crawl requests. This helps respect website rate limits and prevent overwhelming the target website. If not provided, the crawler may use the robots.txt crawl delay if available.
```

Comment:
The correct nomenclature here is `between scrapes`, not `between crawl requests`.